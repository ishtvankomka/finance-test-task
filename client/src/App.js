import './App.scss';
import React from "react";
import Main from './components/main/main';
import { Provider } from "react-redux";
import { store } from './store/store';
import { persistStore } from 'redux-persist'
import { PersistGate } from 'redux-persist/integration/react'
let persistor = persistStore(store)


function App() {

  return (

    <Provider store={store}>
      <PersistGate loading={null} persistor={persistor}>
        <Main />
      </PersistGate>
    </Provider>
  );
}

export default App;