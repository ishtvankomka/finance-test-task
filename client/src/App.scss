* {
    margin: 0;
    padding: 0;
}

$white: #ffffff;
$black: #191414;
$dimed: #333333;

@mixin flexbox($direction, $justify, $align) {
    display: flex;
    flex-direction: $direction;
    justify-content: $justify;
    align-items: $align;
}

body {
    font-family: 'Proxima Nova', sans-serif;
    background-color: $black;
    color: $white;
    @include flexbox(column, none, center);
}
