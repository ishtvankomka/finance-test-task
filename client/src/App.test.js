import renderer from "react-test-renderer";
import App from "./App";

describe('App snapshot test', () => {
    test("it renders a snapshot", async () => {
        const tree = renderer.create(<App />).toJSON();
        expect(tree).toMatchSnapshot();
    });
})
