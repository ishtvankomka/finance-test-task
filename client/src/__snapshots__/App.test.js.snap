// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test it renders a snapshot 1`] = `
<div
  className="main"
>
  <nav>
    <form>
      <div
        className="tickets_list"
      />
      <input
        checked={false}
        id="favorites"
        name="rad"
        onChange={[Function]}
        type="radio"
        value="favorites"
      />
      <label
        className="favorites"
        htmlFor="favorites"
        onClick={[Function]}
      >
        â˜†
      </label>
    </form>
  </nav>
  <div
    className="tickers"
  />
</div>
`;
